docs/src/locators.md

```diff
@@ -1014,6 +1016,55 @@ await Expect(Page
     .ToHaveCountAsync(1);
 ```
 
+The inner locator **must be relative** to the outer locator and is queried starting with the outer locator match, not the document root. Therefore, the following will not work, because the inner locator references the `<ul>` list element outside of the `<li>` list item:
```

Comment:
```suggestion
The inner locator **must be relative** to the outer locator and is queried starting with the outer locator match, not the document root. Therefore, the following will not work, because the inner locator references the `<ul>` list element inside of the `<li>` list item:
```
?